docs/faq.rst

```diff
@@ -0,0 +1,37 @@
+Frequently Asked Questions
+===============================
+
+.. glossary::
+
+	Is the information or data I add open access?
+		No, nothing is open access on CERN Analysis Preservation (except for the projects own source code). It is designed to be a save environment for CERN physicists to use from the very beginning of starting their analysis and at any given moment in its lifetime. It is also meant to increase this lifetime. Access to your analysis is always restricted to your collaboration and the permissions you assign.
+		
+		If you are searching for a service providing open access data or you have some data to share, you may want to check out `http://opendata.cern.ch/ <CERN Open Data>`_.
+
+	With all this rerunning, cloning and reusing - where is my work? Are you trying to automate analyses?
```

Comment:
I would like to know if this is "really" an issue - versus all the other issues that might be out there. Needs feedback/testing with the potential users.